### DISCUSSION TITLE
Should I re-post here issues I recently posted to the original Orgzly github page?

### DISCUSSION THREAD

**Discussion by Keyhabit** [orgzly-revived/orgzly-android-revived#67] [General]
Would this make sense or will the new developers here also take posts on [https://github.com/orgzly/orgzly-android/issues](url) into consideration?

Many thanks!

**Comment by cbkerr** [comment]
See #2 for more discussion

I think we want to refer to both issue lists but you're welcome to continue/move discussion of new issues here. Current efforts are on getting the release channels working, though

**Reply by Keyhabit** [reply]
Okay, I'll see what happens and may re-post here eventually. Thanks for picking up the threads on this project!
